lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfod.cpp

```diff
@@ -114,29 +177,85 @@ SymbolLocator *SymbolLocatorDebuginfod::CreateInstance() {
 
 static std::optional<FileSpec> GetFileForModule(
     const ModuleSpec &module_spec,
-    std::function<llvm::Expected<std::string>(llvm::object::BuildIDRef)>
-        PullFromServer) {
+    std::function<llvm::StringRef(llvm::object::BuildID)> UrlBuilder,
+    bool force_lookup) {
```

Comment:
Should this be more clearly named like `sync_download` to make it clear we want to synchronously download the binary?

Reply:
Yeah. That's definitely more clear.